src/main/java/com/ft/methodearticleinternalcomponentsmapper/transformation/MethodeLinksBodyProcessor.java

```diff
@@ -71,51 +78,40 @@ public MethodeLinksBodyProcessor(ResilientClient documentStoreApiClient, URI uri
     @Override
     public String process(String body, BodyProcessingContext bodyProcessingContext) throws BodyProcessingException {
         if (body != null && !body.trim().isEmpty()) {
```

Comment:
I kept the IllegalStateException in order to avoid wrapping a BodyProcessingException inside a BodyProcessingException.